### PR TITLE
Convert util methods to js (Step #1)

### PR DESCRIPTION
**Pull request by LMS007**
Convert several exposed methods from util.coffee to range-util.js and add a basic set of unit tests for each.

- getTextNodes
- getLastTextNodeUpTo
- getFirstTextNodeNotBefore

This is the first step. I moved a few util methods to a new js file (range-util.js). The next step is to move all the methods out of xpath.coffee and the rest from util.coffee and at least 1 from range.coffee into this new range-util.js file. I'll add basic unit tests as are reasonable but it's hard to guess at all the edge cases for some of these functions so I could use some additional help there.

### CODE REVIEW

File: src/annotator/anchoring/test/range-util-test.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      assert.equal(result.nodeValue, 'text last');
+    });
+
+    it('looks backwards to get the last text node if none are found', () => {
+      container.innerHTML = `<span>text first</span><span>text last</span><div id="too-far"></div>`;
+      const result = getLastTextNodeUpTo(container.querySelector('#too-far'));
+      assert.equal(result.nodeValue, 'text last');
+    });
+
+    it.skip('returns null if no text node exists', () => {
```
Comment (LMS007) [inline src/annotator/anchoring/test/range-util-test.js:66]: These 2 null cases don't seem to be working quite right. I'll give em another look tomorrow
